client/blocks/reader-post-card/index.jsx

```diff
@@ -82,7 +84,7 @@ export default class RefreshPostCard extends React.Component {
 	}
 
 	render() {
-		const { post, site, feed, onCommentClick } = this.props;
+		const { post, site, feed, onCommentClick, showPrimaryFollowButton } = this.props;
```

Comment:
Yes! In the post options menu:

<img width="346" alt="screen shot 2016-11-02 at 12 16 36" src="https://cloud.githubusercontent.com/assets/17325/19911104/3d84ae92-a0f6-11e6-97ae-617e0b2b4651.png">

I thought `showFollowButton` set to `false` might indicate that it was hidden everywhere on the card.